apps/language_server/lib/language_server/source_file.ex

```diff
@@ -334,9 +334,12 @@ defmodule ElixirLS.LanguageServer.SourceFile do
       do: {max(elixir_line - 1, 0), 0}
 
   def elixir_position_to_lsp(urf8_text, {elixir_line, elixir_character}) do
-    utf16_character =
+    line =
       lines(urf8_text)
       |> Enum.at(max(elixir_line - 1, 0))
+
+    utf16_character =
+      (line || "")
       |> elixir_character_to_lsp(elixir_character)
```

Comment:
i really think this is being made less clear due to the pipes
```suggestion
    utf16_character = elixir_character_to_lsp(line || "", elixir_character)
```